logfile.go

```diff
@@ -20,26 +22,44 @@ func lineWorker(die dieCh, r *bufio.Reader, cfg logplexc.Config, sr *serveRecord
 		log.Fatalf("could not create logging client: %v", err)
 	}
 
-	for {
-		select {
-		case <-die:
-			return
-		default:
-			break
-		}
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer watcher.Close()
 
-		l, _, err := r.ReadLine()
-		l = prefix.ReplaceAll(l, []byte(""))
-		if len(l) > 0 {
-			target.BufferMessage(134, time.Now(), "redis",
-				sr.Name, l)
-		}
+	r := bufio.NewReader(f)
 
-		if err != nil {
-			if err == io.EOF {
-				continue
+	go func() {
+		for {
+			select {
```

Comment:
Does it make sense to make "<-die" a select case here that exits the goroutine, e.g. when there's a new config, for example?  Or are we sure that io.EOF will be encountered soon enough in all situations anyway, or something?